src/components/recipes/JSONArgumentsField.js

```diff
@@ -0,0 +1,35 @@
+import { Input } from 'antd';
+import { Map } from 'immutable';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import FormItem from 'console/components/forms/FormItem';
+
+export default class JSONArgumentsField extends React.Component {
+  static propTypes = {
+    disabled: PropTypes.bool,
+    recipeArguments: PropTypes.instanceOf(Map),
+  };
+
+  static defaultProps = {
+    disabled: false,
+    recipeArguments: new Map(),
+  };
+
+  render() {
+    const { disabled, recipeArguments } = this.props;
+
+    return (
+      <div>
+        <p className="action-info">Contextual UI is not available for this action.</p>
+        <FormItem
+          name="arguments"
+          label="JSON blob"
```

Comment:
If the spelling is "Filter Expression" (and not "Filter expression") then this should be "JSON Blob".

Reply:
yup! good catch